app/src/main/res/layout/activity_home.xml

```diff
@@ -113,5 +113,32 @@
         app:layout_constraintStart_toStartOf="parent"
         tools:visibility="visible" />
 
+    <LinearLayout
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/80975e99df78e5de95a541e0dd9de03de1706afa/app/src/main/res/layout/activity_home.xml#L116"></span>This tag and its children can be replaced by one <code>&lt;TextView/&gt;</code> and a compound drawable</td>
    </tr>
  </tbody>
</table>